core/drivers/crypto/caam/caam_desc.c

```diff
@@ -80,12 +80,14 @@ void caam_desc_push(uint64_t *in_entry, paddr_t paddr)
 
 paddr_t caam_desc_pop(uint64_t *out_entry)
 {
-#ifdef CFG_CAAM_BIG_ENDIAN
-	return get_be64(out_entry);
+	const uint32_t *a32 = (const uint32_t *)out_entry;
+#ifdef	CFG_CAAM_BIG_ENDIAN
```

Comment:
A single space between `#ifdef` and `CFG_CAAM_BIG_ENDIAN` is enough.

Reply:
Sure, made the change.

Reply:
Why not `IS_ENABLED(CFG_CAAM_BIG_ENDIAN)`?